integration-tests/grpc-interceptors/pom.xml

```diff
@@ -22,6 +22,11 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-grpc</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.vertx</groupId>
+            <artifactId>vertx-grpc-common</artifactId>
+            <version>4.4.1-SNAPSHOT</version>
```

Comment:
Was it intended to use the snapshot?

Reply:
No that's the mistake. It's a left over of my initial tests.